app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1027,6 +1028,22 @@ class MultipleDisposalsTriageController @Inject() (
                 .whoIsIndividualRepresenting()
             )
 
+          case IncompleteMultipleDisposalsTriageAnswers(
+                Some(Self),
+                None,
+                None,
+                None,
+                None,
+                None,
+                None,
+                None,
+                None
+              ) if isIndividual =>
```

Comment:
an extra guard is needed here and in the single disposal triage controller to prevent the page from being shown on a first return journey - you can look at `fillinOutreturn.isFirstReturn` to help you tell if it's a first return or not